src/template/stats/reviewer-popup.html

```diff
@@ -0,0 +1,51 @@
+<div class="reviewer-popup">
+    <h4 class="text-center">Projects Reviewed</h4>
+
+    <table class="small table table-condensed table-striped">
+        <thead>
+            <tr>
+                <th>Project Name</th>
+                <th>Reviews</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr ng-repeat="review in reviewer['projectReviews']">
```

Comment:
I don't tink two way binding is necessary here. So reviwer list should be one time binding